invenio_communities/searchapp.py

```diff
@@ -18,15 +18,13 @@ def search_app_context():
     return {
         # The Following functions should be defined for the communities
         # search_app_communities_config
-        # search_app_communities_records_config
```

Comment:
can you please check all the comments here and clean them up or report if something is missing?

Reply:
I will create an issue for the one that is missing.